src/components/NewMessage/NewMessage.vue

```diff
@@ -691,14 +690,16 @@ export default {
 		},
 
 		async handleEdit() {
+			const messageId = this.messageToEdit.id
+			const updatedMessage = this.text.trim()
+			this.chatExtrasStore.removeMessageIdToEdit(this.token)
```

Comment:
The store should be purged only after successful edit. Otherwise, edit mode needs to remain on